forge-gui/res/cardsfolder/upcoming/green_suns_twilight.txt

```diff
@@ -0,0 +1,12 @@
+Name:Green Sun's Twilight
+ManaCost:X G
+Types:Sorcery
+A:SP$ DigMultiple | DigNum$ SVar$X/Plus.1 | Reveal$ True | ChangeValid$ Creature,Land | ChangeLater$ True | Optional$ True | RememberChanged$ True | ImprintRest$ True | SubAbility$ DBBranch | SpellDescription$ Reveal the top X plus one cards of your library. Choose a creature card and/or a land card from among them. Put those cards into your hand and the rest on the bottom of your library in a random order. If X is 5 or more, instead put the chosen cards onto the battlefield or into your hand and the rest on the bottom of your library in a random order.
+SVar:DBBranch:DB$ Branch | BranchConditionSVar$ X | BranchConditionSVarCompare$ GE5 | TrueSubAbility$ DBChoose | FalseSubAbility$ Hand | SubAbility$ RestBottom
+SVar:DBChoose:DB$ GenericChoice | Choices$ Battlefield,Hand
+SVar:Battlefield:DB$ ChangeZone | Defined$ Remembered | Origin$ Library | Destination$ Battlefield | NoShuffle$ True | SubAbility$ RestBottom | SpellDescription$ Put the chosen card(s) onto the battlefield
```

Comment:
These other `SubAbility$ RestBottom` shouldn't be needed?

Reply:
Oops, yeah those are artifacts of old version :(